SWLOR.Game.Server/Feature/RecipeDefinition/FabricationRecipeDefinition/FurnitureRecipes.cs

```diff
@@ -687,6 +687,17 @@ private void Tier2()
 				.Component("fine_wood", 4)
 				.Component("ref_scordspar", 2);
 
+			// Bed, Low
```

Comment:
Is the comment wrong or is the recipe type wrong? Is this supposed to be Bed, Low or Bed, Medical Exam?

Reply:
comment